packages/rum/src/boot/recorderApi.spec.ts

```diff
@@ -13,7 +13,7 @@ import type { StartRecording } from './recorderApi'
 import { makeRecorderApi } from './recorderApi'
 
 describe('makeRecorderApi', () => {
-  let setupBuilder: TestSetupBuilder
+  let lifeCycle = new LifeCycle()
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏
```suggestion
  let lifeCycle: LifeCycle
```